management/telemetry/telemetry-dashboards/telemetry-dashboards.md

```diff
@@ -0,0 +1,197 @@
+
+### Active Contract Consumption
+{: .image-popup}
+![Screenshot - KPIs](/management/telemetry/telemetry-dashboards/ac-org-usage-contract.png)
+
+*Active Contract Consumption* shows three different views of PPU consumption in relation to the contractually agreed values. *PPU* is the actual usage per month. The *Predicted PPU* line is a prediction 
+based on the average consumption in the last three months. The *PPU Limit* line is the consumption agreed in the contract, which is determined by allocating the same nemuber of credits to each month of the 
+contract's tenure.
+
+### Consumption
+{: .image-popup}
+![Screenshot - Consumption](/management/telemetry/telemetry-dashboards/ou-consumption.png)
+
+*Consumption* shows three visuals. For the first two, you can drill down by hovering over a point and clicking on it. *PPU Daily* shows daily consumption broken down into groups based on a usage breakdown. 
+*PPU Monthly* shows the same information but aggregated by month. The last visual shows the limits and actual consumption per month.    
+
+### Projects
+{: .image-popup}
+![Screenshot - Projects_1](/management/telemetry/telemetry-dashboards/ou-project1.png)
+
+{: .image-popup}
+![Screenshot - Projects_2](/management/telemetry/telemetry-dashboards/ou-project2.png)
+
+*Projects* show the five most demanding components, configurations, and projects. For these four visuals, you can drill down to see a breakdown of consumption by week. There is also a URL that gives
+direct access to the underlying data. For the five most expensive projects, there is also a usage breakdown, which shows what has consumed the credits. Next listed are the most demanding SQL and non-SQL 
+transformations, which have consumed most of the credits. The last visual is for the most expensive components. 
+
+### Configurations Health
+{: .image-popup}
+![Screenshot - Configuration Health](/management/telemetry/telemetry-dashboards/ou-config-health.png)
+
+*Configurations Health* gives the health of flows and jobs over time. By hovering over a day, you can see more detailed data (the numbers of flows or jobs on that day that ended with success, error, 
+or warning).
+
+## Activity Center
+*Activity Center* provides the most detailed insight into telemetry data from the three modes available. It is not available by default but has to be requested. You need to tell your account 
+manager which project you want to be the target project in your Activity Center. You can have only one target project. Once the Activity Center has been enabled, it will be visible to everyone with access
+to the target project.
+
+{: .image-popup}
+![Screenshot - Activity Centre in KBC](/management/telemetry/telemetry-dashboards/activity-centre-kbc.png)
+
+The *Activity Center* has five tabs, each focused on a different aspect of consumption monitoring. These tabs are available by default for all projects in the organization but can be filtered for specific 
```

Comment:
Above, you say that it is available only for the single target project. Please clarify :-). Thanks.

Reply:
@hhanova that might be my incorrect English... the Activity Center as dashboard is only available in one Keboola project, however, it shows information on consumption and trends for all Keboola project that are for the given customer (organization). :-) thank you.

Reply:
@hhanova, might additional word "allows" on line 82 :)